keyboards/hineybush/h60/keymaps/via/keymap.c

```diff
@@ -3,35 +3,27 @@
  */
 #include QMK_KEYBOARD_H
 
-// Defines names for use in layer keycodes and the keymap
-enum layer_names {
-    _BASE,
-    _FN1,
-    _FN2,
-    _FN3
-};
-
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
-    [_BASE] = LAYOUT_all(
-    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,    KC_EQL,     KC_BSPC,  KC_TRNS,
+    [0] = LAYOUT_all(
```

Comment:
Technically, you can do this too:
```suggestion
    LAYOUT_all(
```